Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -66,15 +66,12 @@ void VideoConfig::Refresh()
     CPUThreadConfigCallback::AddConfigChangedCallback([]() {
       auto& system = Core::System::GetInstance();
 
-      const bool lock_gpu_thread = Core::IsRunningAndStarted();
-      if (lock_gpu_thread)
-        system.GetFifo().PauseAndLock(true, false);
```

Comment:
This change causes dual core to lock up during bootup, because it's waiting for a GPU thread to signal that it's paused but no GPU thread is running yet.

Reply:
https://github.com/dolphin-emu/dolphin/pull/12886